README.md

```diff
@@ -1,3 +1,15 @@
 # devenv-runtime-local
 
 Repo for Local Runtime Configuration
+
+***
+
+> [!WARNING]
+> This repository is deprecated and will be "archived"!
+>
+> It is not used for runtime configuration any longer since releasing:
+> * v0.1.0 of [vehicle-app-python-template](https://github.com/eclipse-velocitas/vehicle-app-python-template) end of May 2023,
+> * v0.1.0 of [vehicle-app-cpp-template](https://github.com/eclipse-velocitas/vehicle-app-cpp-template) begin of June 2023,
+> * v0.2.0 of [devenv-github-workflows](https://github.com/eclipse-velocitas/devenv-github-workflows) end of May 2023.
+>
+> All Velocitas runtime configurations have been moved to [devenv-runtimes](https://github.com/eclipse-velocitas/devenv-runtimes).
```

Comment:
didn't we plan to split the packages again sometimes?

Reply:
If so, would it be possible to archive them now and "de-archive" then again?

Reply:
What's the benefit besides more work? :D

Reply:
To hide the currently unused repos from users - possibly causing confusion at them.
Even if we would not archive them, I would at least place some warning - or at least a hint - that they are currently unused.
I mean this isn't much work and we gain more clarity. Also, don't know if we ever get time for using them again ...

Reply:
Agreed on (at least) deprecation hint in arch sync today